docs/integration_testing.md

```diff
@@ -0,0 +1,30 @@
+# Integration testing ServiceNow Ansible Collection
+
+With the help of integration tests, we make sure that the plugins from the
+ServiceNow Ansible Collection work as expected when we run them against
+a ServiceNow instance. We use `ansible-test integration`, which executes
+our test targets (roles) that contain test playbooks.
+
+A prerequisite for running integration tests is an active ServiceNow
+[Personal Developer Instance](https://developer.servicenow.com/dev.do#!/guides/quebec/developer-program/pdi-guide/personal-developer-instance-guide-introduction)
```

Comment:
Nitpick: Would it be possible to pull the links out of the paragraph. Something like this:

```
[Personal Developer Instance][pdi] (PDI). You can request ...

   [pdi]:
      https://developer.servicenow.com/dev.do#!/guides/quebec/developer-program/pdi-guide/personal-developer-instance-guide-introduction
      (Instructions on obtaining Personal Developer Instance)
```